README.md

```
@@ -231,6 +232,7 @@ The following table shows the supported customizable features for each robot in
 |kr20_r3100| iontec | ✓ | ✓ | ✓ |
 |kr30_r2100| iontec | ✓ | ✓ | ✓ |
 |kr50_r2500| iontec | ✓ | ✓ | ✓ |
+|kr50_r2100| iontec | ✓ | ✓ | ✓ |
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: no_leading_or_trailing; Actual: leading_and_trailing; Unexpected leading pipe

(MD055, table-pipe-style)